tiny-postgresql-event-store/src/main/kotlin/jp/veka/tables/Tables.kt

```diff
@@ -0,0 +1,101 @@
+package jp.veka.tables
+
+import ru.quipy.saga.SagaContext
+class EventRecordTable {
+    companion object {
+        const val name = "event_record"
+        val id = Column(1, "id", String::class)
+        val aggregateTableName = Column(2, "aggregate_table_name", String::class)
```

Comment:
Please check if we have such a column in mongo-based approach. 

Actually that should be a real table name.